Cargo.toml

```diff
@@ -8,6 +8,6 @@ anyhow = "1.0"
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 url = "2.3"
-mysql_async_wasi = "0.31"
+mysql_async_wasi = "=0.31.5"
 hyper_wasi = { version = "0.15", features = ["full"] }
-tokio_wasi = { version = "1", features = ["io-util", "fs", "net", "time", "rt", "macros"] }
+tokio_wasi = { version = "1", features = ["io-util", "fs", "net", "time", "rt", "macros"] }
```

Comment:
Why does this line change?

Reply:
Is the same line... So IDK, but it does not seem to matter